src/components/select-field/SelectField.scss

```diff
@@ -3,6 +3,8 @@
 .select-field {
     .overlay {
         min-width: 100%;
+        max-height: 340px; // show at most 12 items in the dropdown menu
```

Comment:
This component is probably used in lots of places. Would it be worthwhile to add this behavior to a modifier class that can be activated via a prop, defaulted to false?